ex9.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def multiply(array1, array2):
+    """
+    This function multiplies two given arrays of same size element-by-element.
+    :param array1: array of numbers
+    :param array2: array of numbers
+    :return: multiplied arrays of numbers
+    :return: multiplied array of numbers
+    """
+    return np.multiply(array1, array2)
+
+
+if __name__ == "__main__":
+    matrix1 = np.random.randint(9, size=16).reshape(4, 4)
+    matrix2 = np.random.randint(9, size=16).reshape(4, 4)
+    print(multiply(matrix1, matrix2))
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline